### ISSUE TITLE
enhancement: Add AI states of Reolink cameras to iobroker objects

### ISSUE THREAD

**Issue by Percy2Live** [iobroker-community-adapters/ioBroker.synology#253]
The new Reolink cameras offer vehicle, person and pet detection.  So these cameras (model names ending with "a") do not only offer motion detection, but also the mentioned 3 states if the camera detects a person, pet or vehicle.
The current state of these 3 states can easily be received via the reolink http api:
https://www.reddit.com/r/reolinkcam/comments/rd847e/reolink_camera_api_user_guide_dec10_2021/ 

It would be great to not only see the "motion detected" state for these cameras, but also get objects for "person detected", "pet detected" and "vehicle detected", as these states have a much lower false positive rate

**Comment by mcm1957** [comment]
This adpter connects to a synology system and can / will only provide information synology surveillance station supports. Do have any indication taht synology does or will handle those features?

I think this request is misplaced here.

**Comment by Percy2Live** [comment]
Yes, the surveillance station is also processing this information. This is a screenshot I took on my synology:

![image](https://github.com/iobroker-community-adapters/ioBroker.synology/assets/63621052/536fe182-a356-434f-9930-b2a7689091ee)

Enabling these options basically allows to use the AI states as recording trigger events

**Comment by mcm1957** [comment]
Thanks for clarification
